doc/ex/bounding_box.rb

```diff
@@ -1,6 +1,6 @@
 require 'rmagick'
 
-img = Magick::Image.new(200, 200) { self.background_color = '#ffffcc' }
+img = Magick::Image.new(200, 200) { |e| e.background_color = '#ffffcc' }
```

Comment:
I think it would be good to name the block argument more explicitly. Maybe `|info|`, since it's a `Magick::Info` object?